graphql-kickstart-spring-support/src/main/java/graphql/kickstart/spring/AbstractGraphQLController.java

```diff
@@ -107,10 +105,7 @@ public Object graphqlGET(
   }
 
   private Map<String, Object> convertVariablesJson(String jsonMap) {
-    if (jsonMap == null) {
-      return Collections.emptyMap();
-    }
-    return objectMapper.deserializeVariables(jsonMap);
+    return jsonMap != null ? objectMapper.deserializeVariables(jsonMap) : Collections.emptyMap();
```

Comment:
This could also be written as 
```java
return ofNullable(jsonMap).map(objectMapper::deserializeVariables).orElseGet(emptyMap);
```
But I guess that's just a matter of personal preference :)

Reply:
Now I find that your other codes are written as optional.
I think it is better in this case. 
Thank you.